extension/src/setup.ts

```diff
@@ -159,7 +159,7 @@ export const setup = async (extension: IExtension) => {
     return
   }
 
-  extension.setRoots()
+  await extension.setRoots()
```

Comment:
`dvcRoots` are used by the `initialize` called on line 170. If there are multiple workspace folders, I guess it took too long to set before it was used. There must have been some race conditions happening in a basic project as well.